### ISSUE TITLE
User cannot create new account

### ISSUE THREAD

**Issue by baghdadbrian** [StoryMaker/storymaker#170]
When trying to create a new account, with a fresh email and username, StoryMaker returns "Sorry an error occurred."

Log attached (made during attempted account creation)

[bugreport (1).txt](https://github.com/StoryMaker/storymaker/files/435742/bugreport.1.txt)

**Comment by n8fr8** [comment]
This is on a clean install - setting up a Storymaker account?

**Comment by baghdadbrian** [comment]
Yes clean install setting up a new account. I assume it is something on the
website end of things. This has been reported by at least three users, I
also verified personally.

**Comment by n8fr8** [comment]
It has been reported on older builds? Can you test the last known stable and the latest alpha?

**Comment by baghdadbrian** [comment]
I have no idea when the account setup issue began. I am meeting steve later
and will discuss and try to get a handle on when it may have occurred.
Again my assumption is that it is an issue on the storymaker.org side, not
in the app. I will try ti determine whether there is any evidence for that
beyond my feeling. Will check what Steve knows about this.

**Comment by n8fr8** [comment]
Some interesting errors in the log:

"
08-23 11:57:14.625  2086  2400 E cutils  : Failed to mkdirat(/storage/extSdCard/Android): Read-only file system
08-23 11:57:14.625  2086  2400 W Vold    : Returning OperationFailed - no handler for errno 30
08-23 11:57:14.630 27457 27457 W ContextImpl: Failed to ensure directory: /storage/extSdCard/Android/data/org.storymaker.app/files
08-23 11:57:14.630 27457 27457 D ZipHelper: EXPANSION FILE /storage/emulated/0/Android/data/org.storymaker.app/files/g_odvw.main.13.obb FOUND, ADDING TO ZIP
08-23 11:57:14.635  2086  2400 E cutils  : Failed to mkdirat(/storage/extSdCard/Android): Read-only file system"

**Comment by n8fr8** [comment]
3 11:57:21.465 27457 27457 W ContextImpl: Failed to ensure directory: /storage/extSdCard/Android/data/org.storymaker.app/files
08-23 11:57:21.465 27457 27457 D JsonHelper: NEW EXTERNAL DIRECTORY: /storage/emulated/0/Android/data/org.storymaker.app/files/
08-23 11:57:21.465 27457 27457 D CatalogActivity: lang returned from getCurrentLocale: ru
08-23 11:57:21.465  2086  2400 E cutils  : Failed to mkdirat(/storage/extSdCard/Android): Read-only file system
08-23 11:57:21.465  2086  2400 W Vold    : Returning OperationFailed - no handler for errno 30

**Comment by n8fr8** [comment]
Doing real-time testing on build 1106 and found this reported in logcat:
"08-26 02:11:50.403 13718 13718 I chromium: [INFO:CONSOLE(3)] "Uncaught TypeError: Cannot read property 'each' of null", source: https://ajax.cloudflare.com/cdn-cgi/nexp/dok3v=0489c402f5/cloudflare.min.js (3)"

**Comment by n8fr8** [comment]
I thought the cloudflare error might be related to using Orbot/Tor, but even with Orbot disabled/stopped it still occurs.

**Comment by n8fr8** [comment]
I can reproduce the error in my browser, just by going here: https://storymaker.org/accounts/signup/?chrome=0

@vitriolix can someone check the server logs?

**Comment by vitriolix** [comment]
I believe our emailed service went premium and is probably throwing an error and blocking registration.  I will either disable email confirmation or add a new emailer service

**Comment by n8fr8** [comment]
Can we just pay for the service to quickly get it working?

**Comment by baghdadbrian** [comment]
Let me know what the cost is, SWN should be able to cover it in the interim.
